src/commands/repos/UpdateRepos.ts

```diff
@@ -139,6 +123,45 @@ export class UpdateRepos extends AbstractReposCommand {
         Global.isVerbose() && console.log('successfully updated', repoName);
     }
 
+    private async checkoutTagOrBranch(repo: Repository, repoName: string, commit: string, branch: string, tag: string): Promise<void> {
+        let tagToCheckout: string = tag;
+        if (branch.startsWith('release/')) {
+            const currentReleaseVersion: string = branch.substring('release/'.length);
+            Global.isVerbose() && console.log(repoName, `release version: ${currentReleaseVersion}`);
+            if (!tagToCheckout) {
+                try {
+                    tagToCheckout = await repo.getLatestTagOnBranch(`version/${currentReleaseVersion}*`, branch);
+                    Global.isVerbose() && console.log(repoName, `latest tag for release ${currentReleaseVersion}: ${tagToCheckout}`);
+                } catch (e) {
+                    Global.isVerbose() && console.log(repoName, 'no matching tags for release', currentReleaseVersion);
+                }
+            }
+        }
+
+        if (commit) {
```

Comment:
We could theoretically have a release branch and a commit. In that case we would not need to do the above check.